syft/workers/base.py

```diff
@@ -217,8 +220,9 @@ def send_msg(self, msg_type: int, message: str, location: "BaseWorker") -> objec
         """
         if self.verbose:
             print(f"worker {self} sending {msg_type} {message} to {location}")
+
         # Step 0: combine type and message
-        message = (msg_type, message)
+        message = sy.msg.Message(msg_type, message)
```

Comment:
This would look much cleaner if you imported message. That being said, what is this point if this change, why not just keep it a tuple

Reply:
It's a myriad of reasons, mostly organization. I'm working on some more advanced async/protocol/promise functionality and it's become quite annoying that messages aren't an actual type with actual to-strings which help with introspection.

Reply:
eventually Message objects will be passed to send_msg but I didn't want to do too much in one PR. This is an incremental PR for sure.

Reply:
Still would look cleaner importing message

Reply:
Also we should be testing message construction - which we are not.

Reply:
This will also allow us to add functionality inside the messages to make sure they're well formed. At present we only find out if they end up causing an error on the other side.

Reply:
Also this will make the code easier to write - at present it's totally undocumented what goes into a message (aka - our protocol is undocumented). A good first step here is setting out messages types and what information goes in each message.

Reply:
This will also allow us to do smarter simplification/compression in the future

Reply:
So yeah - lots of good reasons I think. :)

Reply:
also - changed it to import Message directly

Reply:
My prefered style is to use:
from sy import msg
and then use msg.Message(...)
That way you always have the source referenced. 
And additionally you follow the google style guide, see http://google.github.io/styleguide/pyguide.html, section 2.2

Reply:
lol - ok - will fix

Reply:
fixed